wallet-new/server/Cardano/Wallet/API/V1/Swagger.hs

```diff
@@ -231,150 +181,223 @@ The number of entries to display for each page. The minimum is **1**, whereas th
-in the URL.
+In order to send money from one of your account to another address, you can create a new
+payment transaction using the `POST /api/v1/transactions` endpoint as follow:
 
-**For the sake of backward compatibility, we expose the legacy version of the API, available simply as
-unversioned endpoints.**
+```
+curl -X POST https://localhost:8090/api/v1/transactions \
+     -H "Content-Type: application/json; charset=utf-8" \
+     -H "Accept: application/json; charset=utf-8"       \
+     --cacert ./scripts/tls-files/ca.crt                \
+     -d '{                                              \
+  "destinations": [{                                    \
+    "amount": 14,                                       \
+    "address": "A7k5bz1QR2...Tx561NNmfF"                \
+  }],                                                   \
+  "source": {                                           \
+    "accountIndex": 0,                                  \
+    "walletId": "Ae2tdPwUPE...8V3AVTnqGZ"               \
+  }                                                     \
+}'
+```
 
-This means that _omitting_ the version number would call the old version of the API. Examples:
+Note that, in order to perform a transaction, you need to have some existing coins on the
+source account! When the transaction succeeds, funds are _moved_ from an address to another.
+Note that, you can at any time see the status of your wallets by using the
+`GET /api/v1/transactions/{{walletId}}` endpoint as follow:
 
 ```
-/api/version
+curl -X GET https://localhost:8090/api/v1/wallets/{{walletId}}?account_index=0  \
+     -H "Accept: application/json; charset=utf-8"                               \
+     --cacert ./scripts/tls-files/ca.crt                                        \
 ```
 
-Compatibility between major versions is not _guaranteed_, i.e. the request & response formats might differ.
+We have here constrainted the request to a specific account, with our previous transaction the
+output should look roughly similar to this:
+
+```json
+{
+    "status": "success",
+    "data": [
+        {
+            "amount": 14,
+            "inputs": [{
+              "amount": 14,
+              "address": "DdzFFzCqrh...fXSru1pdFE"
+            }],
+            "direction": "outgoing",
+            "outputs": [{
+              "amount": 14,
+              "address": "A7k5bz1QR2...Tx561NNmfF"
+            }],
+            "confirmations": 42,
+            "id": "43zkUzCVi7...TT31uDfEF7",
+            "type": "local"
+        }
+    ],
+    "meta": {
+        "pagination": {
+            "totalPages": 1,
+            "page": 1,
+            "perPage": 10,
+            "totalEntries": 1
+        }
+    }
+}
+```
 
-## Response format (V1 onwards)
+Pagination
+----------
 
 **All GET requests of the API are paginated by default**. Whilst this can be a source of surprise, is
```

Comment:
Perfect, as we are already piggybacking on the old skeleton about pagination, we should at least say hint to the reader, at the beginning (see "Creating a new wallet"), this section.